Rakefile

```diff
@@ -1,17 +1,24 @@
 require 'puppetlabs_spec_helper/rake_tasks'
-require 'puppet-lint/tasks/puppet-lint'
-PuppetLint.configuration.send('disable_80chars')
-PuppetLint.configuration.ignore_paths = ['spec/**/*.pp', 'pkg/**/*.pp']
 
-desc 'Validate manifests, templates, and ruby files'
-task :validate do
-  Dir['manifests/**/*.pp'].each do |manifest|
-    sh "puppet parser validate --noop #{manifest}"
-  end
-  Dir['spec/**/*.rb', 'lib/**/*.rb'].each do |ruby_file|
-    sh "ruby -c #{ruby_file}" unless ruby_file =~ %r{spec\/fixtures}
-  end
-  Dir['templates/**/*.erb'].each do |template|
-    sh "erb -P -x -T '-' #{template} | ruby -c"
+desc 'Task to build puppet module and install it'
+task :build do
+  puppet_version = Facter.value(:puppetversion)
+  fail "Can't find a puppet version." if puppet_version.nil?
+  build_cmd = 'puppet module build'
+  uninstall_cmd = 'puppet module uninstall cisco-cisco_aci'
```

Comment:
Should we use --force for uninstall?

Reply:
I haven't had a problem uninstalling without --force. Am I missing something ?

Reply:
I believe that if there are changes in the installed module code it will detect and ask the users to add the --force option.  Try making a change to the installed module code and then try to uninstall without the --force option.

Reply:
Yes, you're right. Added the --force option now.